### DISCUSSION TITLE
Post-Judging QA

### DISCUSSION THREAD

**Discussion by thebrittfactor** [code-423n4/2023-10-party-findings#589] [Announcements]
1. Please post feedback about _specific findings_ on the relevant issue in this repo.
2. More general feedback for the judge can be shared here.

The judge for this contest is @gzeon-c4.

## Reminders

- Please keep your comments as fact-based as possible.
- C4 judges have final authority in determining validity and severity.
- You are welcome to start new discussions here!
- Please link to specific issues wherever relevant.

Thank you!

**Comment by niser93** [comment]
Hi @gzeon-c4 
I left a comment on [#47](https://github.com/code-423n4/2023-10-party-findings/issues/47)

Thanks in advance

**Reply by niser93** [reply]
Added a comment on #47

**Reply by niser93** [reply]
Added a comment on #47. Thank you

**Comment by Henrychang26** [comment]
Thanks for judging @gzeon-c4 
I left a comment on #258

**Reply by Henrychang26** [reply]
@gzeon-c4 I replied to your comment, please take a look again.
Thanks!

**Comment by rvierdiiev** [comment]
hello @gzeon-c4 
i have left comment in #72, #73

**Reply by rvierdiiev** [reply]
@gzeon-c4 pls, look into #72 again, i have added comment

**Comment by zhaojio** [comment]
hello @gzeon-c4 
I have left comment in #257 #356 #152 #286

**Comment by klau5dev** [comment]
Hello, @gzeon-c4 
I have left comment in #459

**Comment by hungdoo** [comment]
Hi @gzeon-c4 ,
Thanks for judging.
I've left a comment in https://github.com/code-423n4/2023-10-party-findings/issues/399

**Comment by pavankv241** [comment]
Hi @gzeon-c4 ,
I've left a comment on #359

**Comment by KupiaSecAdmin** [comment]
Hi @gzeon-c4 
Thanks for your prompt judging.
I've left a comment in #414.

**Comment by JeffCX** [comment]
left comments on https://github.com/code-423n4/2023-10-party-findings/issues/376

**Reply by JeffCX** [reply]
left comments on 

https://github.com/code-423n4/2023-10-party-findings/issues/348

**Reply by JeffCX** [reply]
left more comments on #414, #348 is clearly a duplicate of #100 and #414 

left more comments on #376

**Comment by TradMod** [comment]
Thanks @gzeon-c4 for your judging sir.
Left a comment https://github.com/code-423n4/2023-10-party-findings/issues/233#issuecomment-1822097564

**Comment by Emedudu** [comment]
Hello @gzeon-c4 
Thanks for your effort🫡
I left comments in: #534 , #511 , #493 , #529

**Reply by Emedudu** [reply]
Hi @gzeon-c4 

Please check #534, #493 . I made another comment

**Reply by Emedudu** [reply]
@gzeon-c4 , Just dropped another comment #534

**Reply by Emedudu** [reply]
@gzeon-c4  #522 🙏

**Reply by Emedudu** [reply]
@gzeon-c4 Please check #522 again

**Comment by Topmark1** [comment]
Thanks @gzeon-c4 for your great effort in judging sir.
Left a comment in https://github.com/code-423n4/2023-10-party-findings/issues/290

**Reply by Topmark1** [reply]
and https://github.com/code-423n4/2023-10-party-findings/issues/154

**Comment by 0xStalin-eth** [comment]
Hello @gzeon-c4 , great work judging this contest, I added a comment on #533 , please take a look and let me know your thoughts about it.

**Reply by 0xStalin-eth** [reply]
Added a comment on #515

**Reply by 0xStalin-eth** [reply]
Added a comment on #142

**Comment by Ch-301** [comment]
I left comment on #387

**Comment by rodiontr** [comment]
Hi @gzeon-c4 ! Thanks for judging the issues! Left a comment on #172 and #145

**Reply by rodiontr** [reply]
Left more comments on #145

**Reply by rodiontr** [reply]
Left comments on #172 again

**Reply by rodiontr** [reply]
@gzeon-c4 Would really appreciate if you take the final look at #414 and #145

**Comment by 0xdeth** [comment]
Hi @gzeon-c4 

All the comments I've left are on behalf of TresDelinquentes.

#233, #311, #533, #351, #163, #167

Cheers and thank you for the super fast judging.

**Reply by 0xdeth** [reply]
Hey @gzeon-c4 

I left another comment on #163.

Cheers.

**Reply by 0xdeth** [reply]
Hey @gzeon-c4 

Left another comment on #163.

**Reply by 0xdeth** [reply]
Added a comment on #237.

**Comment by adeolu98** [comment]
Hi @gzeon-c4, thanks for the swift judging. 

I left comments on #66 , #504  and #140

**Comment by J4X-Security** [comment]
Hi @gzeon-c4, thanks for judging.

I left a comment on #453.

I hope this aides in understanding the differentiation between my submission and the described issue in the comment.

**Reply by J4X-Security** [reply]
Also commented on #463 and #480 please also take a look

**Comment by Shubh0412** [comment]
Hello @gzeon-c4 
Thank you for swift judging.
Left a comment at #309

**Comment by thangtranth** [comment]
Hi @gzeon-c4 , 

I left comments on #314 and #318 . Thanks for judging.

**Comment by Bauchibred** [comment]
Hi @gzeon-c4, thanks for judging! Left comments on #262 & #580.

**Comment by acestraightm** [comment]
Hi, @gzeon-c4, Thanks for judging! Left a comment on [#219](https://github.com/code-423n4/2023-10-party-findings/issues/219).

**Comment by radeveth** [comment]
###

Hello, @gzeon-c4!

I left comments on:
-> #120 
-> #122 
-> #126 
-> #128 
-> #130 
-> #136 

####

Thank you for the quick judging!

**Reply by radeveth** [reply]
Hello, @gzeon-c4!

I also left comments on #304 and #123.

**Reply by radeveth** [reply]
Hello, @gzeon-c4!

I left another comments on #120, #126, #128, #130, #304.

###

Have a nice day!

**Reply by gzeon-c4** [reply]
I will consolidate my replies here since they seems to be surrounding the same misunderstanding. Contest [scope](https://github.com/code-423n4/2023-10-party/blob/b23c65d62a20921c709582b0b76b387f2bb9ebb5/README.md?plain=1#L93) stated that:
> - Authorities have “root user privileges” over a Party and is it well aware that they can exploit or break Parties in a wide variety of ways, especially if they are set to a malicious EOA or unsafe smart contract. This is why authorities are almost always smart contracts that are highly audited and trusted, and are NOT expected to be EOAs.

Responsibility of making sure all the interaction from the Authority to the Party is safe and does not break any invariant falls on the Authority contract `(e.g. well aware that they can exploit or break Parties in a wide variety of ways)`. Audit of the Authority contract should consider it side effect and it should not lead to bad state in the Party contract `(e.g. are highly audited and trusted)`

**Reply by radeveth** [reply]
Hey, @gzeon-c4 

Would really appreciate if you take look at my comments on this issues (especially the final one) and tell me why my issues are invalid because they are not about malicious authorities.

**Reply by radeveth** [reply]
especially #126 and #128

---

@gzeon-c4

**Reply by radeveth** [reply]
#120, #126, #128, #130, #304

Would really appreciate!

Have a good one!

**Comment by JeffCX** [comment]
Does the appeal period applies to this content?

**Comment by acestraightm** [comment]
hi, @gzeon-c4.

I just left a comment on [#220](https://github.com/code-423n4/2023-10-party-findings/issues/220)

**Comment by Topmark1** [comment]
hi, @gzeon-c4
left a concluding comment on https://github.com/code-423n4/2023-10-party-findings/issues/154

**Comment by osmanozdemir1** [comment]
Hi, @gzeon-c4 
Thanks for all of your efforts during this contest.
Left comments on #319 and #394
I would appreciate if you could take a look when you have time.
Thanks.

**Reply by osmanozdemir1** [reply]
I just saw you already answered #319. 
Thanks for that.

**Comment by Henrychang26** [comment]
@gzeon-c4 
I replied to your comment on #258

**Comment by 0xStalin-eth** [comment]
@gzeon-c4  I added a comment on #534 , please take a look and let me know what you think about it

**Reply by 0xStalin-eth** [reply]
I added a comment on #529, please take a look

**Comment by aktech297** [comment]
Hi,

Apologies for missing the time to appeal.

We left comments on

#401
#432

Kindly share your thoughts.

**Comment by gzeon-c4** [comment]
hey everyone, will be resolving the remaining comments over the weekend

**Comment by Emedudu** [comment]
Hello  @gzeon-c4 ,

Please check #522

**Comment by 0xdeth** [comment]
Final thoughts on #163?

**Comment by radeveth** [comment]
Hey, @gzeon-c4!

I left some final thoughts on #126 and #128. Please take a look.

Have a good one!
